ts/features/fims/components/FimsSuccessBody.tsx

```diff
@@ -68,7 +69,7 @@ export const FimsFlowSuccessBody = ({ consents }: FimsSuccessBodyProps) => {
         <Label weight="Regular">
           {I18n.t("FIMS.consentsScreen.subtitle2")}
         </Label>
-        <Label weight="Bold">{consents.redirection.display_name}.</Label>
+        <Label weight="Bold">{consents.redirect.display_name ?? ""}.</Label>
```

Comment:
I think that in this situation we should handle the copy more like 
"per la tua autenticazione ${displayName? `su ${displayName}` : "." }" -- with correct I18n handling, ca va sans dire -- 

what do you think? after all this feature still does have some rough edges, so we could just open a task for this and think about it in the future, but it has to be addressed nonetheless imo

Reply:
I would not work on this at the moment, since it probably is just a wrong specification so it will change sooner. I suggest we leave it as it is